python/GafferImageTest/OpenImageIOReaderTest.py

```diff
@@ -169,9 +169,8 @@ def testNoOIIOErrorBufferOverflows( self ) :
 		n = GafferImage.OpenImageIOReader()
 		n["fileName"].setValue( "thisReallyReallyReallyReallyReallyReallyReallyReallyReallyLongFilenameDoesNotExist.tif" )
 
-		for i in range( 0, 300000 ) :
-			with IECore.IgnoredExceptions( Exception ) :
-				n["out"]["dataWindow"].getValue()
+		with IECore.IgnoredExceptions( Exception ) :
```

Comment:
If you're sure it's safe to remove this test, then let's just remove it completely, because without the loop it's useless. It was guarding against a scenario where we never retrieved the error message from the OIIO cache, and multiple errors just accumulated and then went over an internal limit...